experiments/golden-results/muen-summary.txt

```diff
@@ -3562,8 +3557,13 @@ Occurs: 1 times
 +===========================GNAT BUG DETECTED==============================+
 | GNU Ada (ada2goto) Constraint_Error Symbol_Table_Info.Symbol_Maps.Constant_Reference: key not in map|
 Error detected at REDACTED
+--
+Occurs: 4 times
+<========================>
+raised SYSTEM.ASSERTIONS.ASSERT_FAILURE : failed postcondition from tree_walk.ads:84
+
```

Comment:
Higher up you removed 3 instances of Do_Type_Definition / Unknown expression kind / N_Access_Function_Definition .  Here there are 4 extra assertion failures.  Forgive me but it doesn't look like this is a net improvement.  Also, where is the extra error coming from?